app.js

```diff
@@ -84,7 +84,7 @@ Vue.createApp({
           case "z-a":
             return b.name.localeCompare(a.name)
           case "owned":
-            return a.owned === b.owned ? a.level.localeCompare(b.level) : a.owned - b.owned
+            return a.owned === b.owned ? a.name.localeCompare(b.level) : a.owned - b.owned
```

Comment:
```suggestion
            return a.owned === b.owned ? a.name.localeCompare(b.name) : a.owned - b.owned
```
Shouldn't be name for `b` too ?

Reply:
Oh, yeah. It should.